core/iwasm/compilation/aot_llvm.c

```diff
@@ -1474,18 +1494,10 @@ aot_create_comp_context(AOTCompData *comp_data, aot_comp_option_t option)
     LLVMTargetDataRef target_data_ref;
 
     /* Initialize LLVM environment */
-#if WASM_ENABLE_LAZY_JIT != 0
     LLVMInitializeCore(LLVMGetGlobalPassRegistry());
     LLVMInitializeNativeTarget();
     LLVMInitializeNativeAsmPrinter();
     LLVMInitializeNativeAsmParser();
-#else
-    LLVMInitializeAllTargetInfos();
-    LLVMInitializeAllTargets();
-    LLVMInitializeAllTargetMCs();
-    LLVMInitializeAllAsmPrinters();
-    LLVMLinkInMCJIT();
```

Comment:
Had better add these lines except LLVMInitializeAllTargetMCs(), they may be required by wamrc when generating aot file for all targets, e.g. riscv, arm, xtensa.